app/components/SelectSpecies/index.js

```diff
@@ -66,15 +66,15 @@ const SelectSpecies = () => {
         species: [
           {
             id: specie.guid,
-            aliases: specie.scientificName,
+            aliases: specie.aliases,
             treeCount: 1,
           },
         ],
       });
     } else {
       let updatedSampleTrees = [...inventory.sampleTrees];
       updatedSampleTrees[inventory.completedSampleTreesCount].specieId = specie.guid;
-      updatedSampleTrees[inventory.completedSampleTreesCount].specieName = specie.scientificName;
+      updatedSampleTrees[inventory.completedSampleTreesCount].specieName = specie.aliases;
```

Comment:
Sorry, I can't see where this applies in the UI? I always see the scientific name in my tests everywhere I work with species.

Reply:
My fault. Got it.